<h1>Data Science Tools and Ecosystem</h1>

In this notebook, Data Science Tools and Ecosystem are summarized.

Some of the popular languages that Data Scientists use are:

1. Python
2. R
3. SQL
4. Julia
5. Scala


Some of the commonly used libraries used by Data Scientists include:

1. Pandas
2. NumPy
3. Matplotlib
4. Scikit-learn
5. TensorFlow


| Data Science Tools |
|--------------------|
| Jupyter Notebook   |
| RStudio            |
| Apache Zeppelin    |


 ### Below are a few examples of evaluating arithmetic expressions in Python.

      1. Addition
      2. Subtraction
      3. Multiplication
      4. Division
      5. Exponentiation
      6. Modulus

In [1]:
# This is a simple arithmetic expression to multiply then add integers
(3*4) + 5


17

In [2]:
# This will convert 200 minutes to hours by dividing by 60
200 / 60


3.3333333333333335

**Objectives:**

- List popular languages for Data Science
  - Python
  - R
  - SQL

- List commonly used libraries in Data Science
  - Pandas
  - NumPy
  - Matplotlib

- Create a table of Data Science tools

  | Data Science Tools     |
  |------------------------|
  | Jupyter Notebook       |
  | RStudio                |
  | Apache Zeppelin        |

- Show examples of evaluating arithmetic expressions

  - Addition: `3 + 5`
  - Subtraction: `10 - 4`
  - Multiplication: `7 * 6`
  - Division: `8 / 2`
  - Exponentiation: `2 ** 3`
  - Modulus: `10 % 3`

- Convert minutes to hours
  - 200 minutes to hours: `200 / 60 = 3.33`


## Author
Emrah Kızıltan
